src/opendr/perception/facial_expression_recognition/landmark_based_facial_expression_recognition/algorithm/datasets/landmark_extractor.py

```diff
@@ -0,0 +1,102 @@
+# you may not use this file except in compliance with the License.
```

Comment:
I could be wrong, but has part of this license/comment been chopped off by mistake by any chance? Appears odd to start like this and missing the years.

Reply:
Yes it was cut by mistake and it is corrected now. Thank you for the comment.